python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -567,15 +568,24 @@ def test_delete_date_range(arctic_library):
     assert lib["symbol"].version == 1
 
 
+@pytest.mark.parametrize("pymongo_patcher", ["disable_pymongo_in_adapter", None])
 @pytest.mark.skipif(not RUN_MONGO_TEST, reason="Mongo test on windows is fiddly")
-def test_mongo_repr(mongo_test_uri):
+def test_mongo_repr(request, pymongo_patcher, mongo_test_uri):
```

Comment:
I think it's a bit cleaner to just have two separate tests, rather than the parameterized fixture, or trying to shoe-horn `monkeypatch` in to a fixture. And we should assert that when pymongo is installed, we get the good exception. This test doesn't belong in `test_mongo_repr`.

Reply:
Addressed